### ISSUE TITLE
How to Save Text And Pencil Written Text 

### ISSUE THREAD

**Issue by devaliy** [stephanrauh/ngx-extended-pdf-viewer#1772]
Is there a way to capture or save the text written on the pdf so that it doesn’t disappear when you reopen the file

**Comment by Blunderchips** [comment]
Duplicate of https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/1679? If so, not yet (or at least directly).

**Comment by stephanrauh** [comment]
@devaliy When I read your issue, I had the same idea as @Blunderchips. I don't know exactly what a "text written on the PDF" is, but my best guess is you're referring to the new PDF editor feature. Is this correct? If it's not, can you rephrase your question more precisely?

**Comment by TechOffeiceAhmad** [comment]
is there any way to save pdf after sign on it ?

**Comment by stephanrauh** [comment]
That's precisely the topic of #1679. Maybe it's already solved when I implement #1685. When you use the "download" button, the contents the user has added using the editor is part of the downloaded file, so I assume it will also be there when I implement the programmatic API.

**Comment by stephanrauh** [comment]
Holy moly... it was a bug, not a missing feature! Please update to version 17.0.0-beta.1. It includes the editor annotations both in print and in the download.
